<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTACData</class>
 <widget class="QWidget" name="QTACData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symbol Data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/images/icons/TIFF/back_alt.tiff</normaloff>:/icons/images/icons/TIFF/back_alt.tiff</iconset>
  </property>
  <widget class="QToolButton" name="upToolButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/ChartGeany.qrc">
     <normaloff>:/png/images/icons/PNG/up_alt.png</normaloff>:/png/images/icons/PNG/up_alt.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="downToolButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../resources/ChartGeany.qrc">
     <normaloff>:/png/images/icons/PNG/down_alt.png</normaloff>:/png/images/icons/PNG/down_alt.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEditData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>500</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>abced
xyz
</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources/ChartGeany.qrc"/>
 </resources>
 <connections/>
</ui>
